modules/ROOT/pages/deployment/services/tls.adoc

```diff
@@ -0,0 +1,101 @@
+= Transport Layer Security (TLS)
+:toc: macro
+
+:description: TLS is important to secure communication between parties. Infinite Scale provides TLS for internal communication and also for communication with users or services which are not provided by Infinite Scale. 
+
+include::partial$deployment/services/beta-statement.adoc[]
+
+// remove the remark to get the toc rendered if there are more than one headlines
+{empty} +
+
+toc::[]
+
+== Introduction
+
+{description} While focusing on describing the environment variables and their relationships, managing certificates is not the scope of this document.
+
+If not explicitly mentioned otherwise, the environment variables for services start with the environment variable name. Names starting with `OCIS_` generally have a global scope and are not bound to a particular service. 
+
+== Secure Service-To-Service Communication
+
+When deploying a distributed setup, it is recommended to secure communication and authentication between services.
+
+Most services currently provide TLS for service-to-service communication and authentication with the exception of the xref:{s-path}/frontend.adoc[frontend] and the xref:{s-path}/ocdav.adoc[ocdav] service. TLS will be added to the latter ones in a future release.
+
+NOTE: TLS is usually defined per instance on a global scope, though you can define the settings per service too.
```

Comment:
```suggestion
[NOTE]
====
* TLS is usually defined per instance on a global scope, though you can define the settings per service too.
* When setting TLS modes to _insecure_, the traffic can't be sniffed anymore, but it would be still possible to intercept it.
====
```